arch/arm/core/aarch32/irq_relay.S

```diff
@@ -26,10 +26,7 @@
 
 _ASM_FILE_PROLOGUE
 
-/* When CONFIG_BOOTLOADER_MCUBOOT is set the firmware image
- * is chain-loadable and does not need to forward interrupts.
- */
-#if defined(CONFIG_SW_VECTOR_RELAY) && !defined(CONFIG_BOOTLOADER_MCUBOOT)
+#if defined(CONFIG_SW_VECTOR_RELAY)
```

Comment:
Is it possible to completely remove this #ifdef and make this conditional in CMakeLists?

Reply:
actually, i see that irq_relay.S is already compiled-in only when SW_VECTOR_RELAY is set

Reply:
Done.

Reply:
sounds reasonably ^^